packages/theme/components/CartSidebar.vue

```diff
@@ -141,12 +161,20 @@ export default {
     const products = computed(() => cartGetters.getItems(cart.value));
     const totals = computed(() => cartGetters.getTotals(cart.value));
     const totalItems = computed(() => cartGetters.getTotalItems(cart.value));
-
+    const { wishlist, addItem: addItemToWishlist, isInWishlist } = useWishlist();
+    const isWishlistDisabled = computed(() => wishlistGetters.isWishlistDisabled(wishlist.value));
     const updateQuantity = debounce(async ({ product, quantity }) => {
       console.log('debug:updateQuantity', product, quantity);
       await updateItemQty({ product, quantity });
     }, 500);
 
+    const handleSaveForLaterClick = async(product) => {
+      if (!isInWishlist({product})) {
```

Comment:
How about flattening the code using "return early" pattern here? We don't care if a product is on the wishlist, so we could exit from the function as soon as possible.

Reply:
I think that the readability will remain the same.